### ISSUE TITLE
Screen Reader does not work inside VPN on Linux

### ISSUE THREAD

**Issue by ValentinaPC** [mozilla-mobile/mozilla-vpn-client#717]
#### Affected versions:
* v. 2.9.0; 

#### Affected Platforms:
* Linux Ubuntu 20.04;

#### Prerequisites:
* Have a FxA account with active subscription for Mozilla VPN;

#### Steps to reproduce:
1. Open the VPN;
1. Turn on Narrator or Screen Reader;
1. Focus inside the VPN app;

#### Expected result:
* The Screen Reader works inside the VPN.

#### Actual result:
* The Screen Reader does not work inside the app;

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-122)
┆Reporter: Data Sync

**Comment by lesleyjanenorton** [comment]
Also not working on Mac anymore. After some digging, I'm pretty sure this stopped working (on Mac anyway) after the move to Qt 5.14. I'm not sure why yet though.

**Comment by lesleyjanenorton** [comment]
@ValentinaPC I can't reproduce this with the latest Windows build. Are you still encountering this?

**Comment by lesleyjanenorton** [comment]
Closing as fixed since I can't reproduce this on Windows with Narrator or on Mac with VoiceOver. Please reopen if you find this is still an issue!

**Comment by ValentinaPC** [comment]
I am still reproducing: clicking the VPN window is recorded by Narrator, but actions inside the app, no.
See video below.

https://user-images.githubusercontent.com/15685960/114829448-c1cc8b00-9dd3-11eb-9531-672a7c67ffd8.mp4

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

Valentina Virlics could you verify that this is still an isue with Qt6 builds? I seem to remember a conversation where this was limited to Qt5 builds but perhaps that is wishful thinking. Thank you!

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Lesley Norton We do not yet have Windows QT6 builds, or if we have QA does not know about them or where to download.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this on the last VPN 2.8.0 (2.202203162042) build, while suing Win 10.
QT6 version: 6.2.3;

The Narrator still does not work inside the app.

**Comment by lloan** [comment]
@lesleyjanenorton I was able to reproduce on Windows. It just selects the whole app and the only thing the narrator says is "Mozilla VPN" - nothing else. 

<img width="314" alt="image" src="https://user-images.githubusercontent.com/3924990/163601828-5321f5f9-c305-4514-a684-c9893d87adfe.png">

**Comment by lloan** [comment]
@ValentinaPC Do you happen to know if this happens on any other platform or is it only Windows?

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Screen reader seems to work accordingly on Mac OS. So, as mentioned in the issue, currently we have issues only on Windows and  Linux.

Lloan Alas

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Verified this on Windows 10, while using Mozilla VPN 2.9.0 (2.202205291610) build.

The Narrator is working as expected inside the Mozilla VPN.

Will leave this open until it can be checked on Linux as well. (https://mozilla-hub.atlassian.net/browse/VPN-2268?searchSessionId=f085d001-b5e8-441d-80ca-29b1192b5efc&searchObjectId=274720&searchContainerId=10145&searchContentType=issue ( https://mozilla-hub.atlassian.net/browse/VPN-2268?searchSessionId=f085d001-b5e8-441d-80ca-29b1192b5efc&searchObjectId=274720&searchContainerId=10145&searchContentType=issue|smart-link ) )

**Comment by data-sync-user** [comment]
➤ Maria Berlinger commented:

Hello,

This is still reproducible using Mozilla VPN 2.9.0 (2.9.0~rc49-focal1) on Ubuntu 20.04. The screen reader doesn’t work inside the app.

**Comment by data-sync-user** [comment]
➤ Lesley Norton commented:

thank you Maria Berlinger Valentina Virlics . Updating the ticket title and labels for clarity.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Affects a subset of the 3% of users on Linux, so moving to non-blocker. Not reassigning to QA though as I don’t believe there is any reason to believe this is now fixed.

**Comment by data-sync-user** [comment]
➤ Valentina Virlics commented:

Currently, while using Mozilla VPN 2.21 RC, the screen reader focuses the app. Other wrong behaviors are tracked in specific issues. (i.e. https://mozilla-hub.atlassian.net/browse/VPN-6193 ( https://mozilla-hub.atlassian.net/browse/VPN-6193|smart-link ) )
